features/install.feature

```diff
@@ -78,6 +78,29 @@ Feature: Installation
       And the file "config/initializers/localeapp.rb" should contain "config.api_key = ENV['LOCALEAPP_API_KEY']"
       And the exit status should be 0
 
+  Scenario: Running install with .env
```

Comment:
Please indicate that ".env" is an existing file, not a command line
argument or option as we might understand it (like `--env` switch),
especially because all other scenarios in this feature deal with
arguments given to the `install` command.